pac4j-oidc/src/main/java/org/pac4j/oidc/profile/azuread/AzureAdProfile.java

```diff
@@ -46,4 +66,33 @@ public String getUsername() {
         return (String) getAttribute(AzureAdProfileDefinition.UPN);
     }
 
+    public String getAccessTokenFromRefreshToken() {
```

Comment:
Currently the String that is returned is json with attributes of:
token_type, scope, expires_in, ext_expires_in, expires_on, not_before, resource, access_token, refresh_token

Should it be left as is, or return some sort of AzureAdResponse ?

Reply:
In fact, I missed the point here but this method should be in the `AzureAdClient`, taking an `AzureAdProfile` as input and returning only an access token.

Reply:
How should the end user get a reference to the AzureAdClient ?

Reply:
The `AzureAdClient` has been defined in the `Config`  component, so `config.findClient("AzureAdClient")` will do the job...

Reply:
I don't think that is 100% correct....Here is the (verbose) gist as I see it:

`
org.pac4j.core.config.Config config = org.pac4j.core.config.ConfigSingleton.getConfig();
                org.pac4j.core.client.Clients clients = config.getClients();
                org.pac4j.oidc.client.AzureAdClient azureAdClient = clients.findClient( org.pac4j.oidc.client.AzureAdClient.class );
String accessToken = azureAdClient.getAccessTokenFromRefreshToken( azureAdProfile );
`

Or the one liner gist if you prefer:

`String accessToken = org.pac4j.core.config.ConfigSingleton.getConfig().getClients().findClient( org.pac4j.oidc.client.AzureAdClient.class ).getAccessTokenFromRefreshToken( azureAdProfile );`

If you approve this gist I will open up a second Pull Request.

Reply:
In fact, you don't need to open a second PR: just move the `getAccessTokenFromRefreshToken` method from the `AzureAdProfile` to the `AzureAdClient`. Thanks